doc/2/controllers/models/get-device/index.md

```diff
@@ -16,7 +16,7 @@ Gets an device model.
 ### HTTP
 
 ```http
-URL: http://kuzzle:7512/_/device-manager/models/device/:_id
+URL: http://kuzzle:7512/_/device-manager/models/device/:model
```

Comment:
Actually, the slug `_id` was okay. [See this](https://github.com/kuzzleio/kuzzle-device-manager/blob/master/lib/modules/model/ModelsController.ts#L49C52-L49C62)